sdkit/models/model_loader/realesrgan.py

```diff
@@ -1,30 +1,49 @@
 import torch
 import os
+
 from basicsr.archs.rrdbnet_arch import RRDBNet
 from realesrgan import RealESRGANer
 
 from sdkit import Context
 
+
 def load_model(context: Context, **kwargs):
+    """ Load the model. """
     model_path = context.model_paths.get('realesrgan')
+    if not model_path:
+        raise ValueError('model_path not found')
 
     RealESRGAN_models = {
-        'RealESRGAN_x4plus': RRDBNet(num_in_ch=3, num_out_ch=3, num_feat=64, num_block=23, num_grow_ch=32, scale=4),
-        'RealESRGAN_x4plus_anime_6B': RRDBNet(num_in_ch=3, num_out_ch=3, num_feat=64, num_block=6, num_grow_ch=32, scale=4)
+        'RealESRGAN_x4plus': RRDBNet(
+            num_in_ch=3, num_out_ch=3, num_feat=64,
+            num_block=23, num_grow_ch=32, scale=4),
+        'RealESRGAN_x4plus_anime_6B': RRDBNet(
+            num_in_ch=3, num_out_ch=3, num_feat=64,
+            num_block=6, num_grow_ch=32, scale=4)
     }
 
     model_to_use = os.path.basename(model_path)
     model_to_use, _ = os.path.splitext(model_to_use)
     model_to_use = RealESRGAN_models[model_to_use]
 
     half = context.half_precision if context.device != 'cpu' else False
-    model = RealESRGANer(device=torch.device(context.device), scale=4, model_path=model_path, model=model_to_use, pre_pad=0, half=half)
+    model = RealESRGANer(
+        device=torch.device(context.device),
+        scale=4, model_path=model_path, 
+        model=model_to_use, pre_pad=0, 
+        half=half
+    )
     if context.device == 'cpu':
         model.model.to('cpu')
 
     model.model.name = model_to_use
 
     return model
 
-def unload_model(context: Context, **kwargs):
-    pass
+
+def unload_model(_: Context, **__):
+    """
+        Unload the model from the context.
+
+        # TODO: Implement this?
```

Comment:
This stub is only required if the module has some custom tasks to perform, before the cleanup. In this case, realesrgan has nothing extra to do. Hence the `pass` statement.

And these modules (`realesrgan`, `stable-diffusion` etc) are internal modules. They aren't exposed to the user-facing API, so I don't think we need to put redundant documentation in every internal module, when it's clear what they're doing.

Each sub-module inside `model_loader` implements the API of defining: `load_model()` and `unload_model()`. These functions are quite clear from their name, so we don't need redundant documentation for it.